astropy/coordinates/representation.py

```diff
@@ -1583,8 +1614,9 @@ def represent_as(self, other_class, differential_class=None):
         # is passed in, using the ._re_represent_differentials() method
         if inspect.isclass(other_class) and not differential_class:
             if issubclass(other_class, PhysicsSphericalRepresentation):
-                return other_class(phi=self.lon, theta=90 * u.deg - self.lat, r=1.0,
-                                   copy=False)
+                return other_class(
```

Comment:
Why the change? Since this going to be changed in #11470, we can avoid some rebase hassle by keeping it as is...

Reply:
just a holdover as I was rolling back parts of the PR.

Reply:
✅